config/image.go

```diff
@@ -35,6 +35,9 @@ type ImageConfig struct {
 	// Dockerfile The path to the ``Dockerfile`` used to build the image. This
 	// path is relative to the **context**.
 	Dockerfile string
+	// Dobifile is a in house version of a dockerfile
+	// type: mapping ``[Valid Docker command]: value``
+	Steps []map[string]string
```

Comment:
I agree, `Steps` sounds like a good name for this.  The comment just needs to be updated.

"Steps Inline Dockerfile steps to use instead of a Dockerfile"